src/components/Map.js

```diff
@@ -1,20 +1,20 @@
 import React from "react";
-import { connect } from 'react-redux';
+import { connect } from "react-redux";
 import mapboxgl from "mapbox-gl";
-import { initializeProviders, toggleProviderVisibility } from '../actions';
+import { initializeProviders, toggleProviderVisibility } from "../actions";
+import "../map.css";
 import MapboxGeocoder from "@mapbox/mapbox-gl-geocoder";
-// import { insertPopup, Popup } from "./PopUp.js";
-
-import '../map.css';
+import "../index.css";
+import * as turf from "@turf/turf";
 
 mapboxgl.accessToken =
   "pk.eyJ1IjoicmVmdWdlZXN3ZWxjb21lIiwiYSI6ImNqZ2ZkbDFiODQzZmgyd3JuNTVrd3JxbnAifQ.UY8Y52GQKwtVBXH2ssbvgw";
 
 class Map extends React.Component {
-  reflectProviderVisibility = (type) => {
-    const visibility = type.visible ? 'visible' : 'none';
-    this.map.setLayoutProperty(type.id, 'visibility', visibility);
-  }
+  reflectProviderVisibility = type => {
+    const visibility = type.visible ? "visible" : "none";
```

Comment:
We all installed Prettier as a group at some point and these were the settings -- I have no strong feelings and am happy to go whichever way.

Reply:
i don't have strong feelings either personally. I hadn't heard of the team choosing any particular tool (is there some documentation on how the team was linting the code using Prettier? I've been using single quotes so far and nobody has said anything), all I've seen is the PR in #39. Maybe it makes sense to resolve code formatting in that PR instead of here, and accept this PR with either format.